tests/unit/TodoList1.spec.ts

```diff
@@ -1143,7 +1144,6 @@ describe('TodoList - UI - BCOMPs', () => {
     expect(wrapper.find('.list-item__actions .v-btn').attributes('disabled')).toBe('disabled')
     expect(button.querySelector('.v-btn__content').textContent).toContain('File Annual Report')
 
-    // store.state.isCoaPending = false // reset for future tests
```

Comment:
Is it OK for this getter do not be reset, or should it be un-mocked?

Reply:
would `jest.restoreAllMocks()` in `afterAll()` work for resetting?

Reply:
Okay. I will check this again

Reply:
The value of the said getter does not seem to matter for other tests resetting the value at the end of this test to be safe

For ref: https://jestjs.io/docs/jest-object#jestrestoreallmocks

Reply:
Looks good. Probably we'll use this function in the future to clean up mocks at the end of a test.